drivers/openstack/openstack.go

```diff
@@ -516,6 +517,27 @@ func (d *Driver) resolveIds() error {
 		}).Debug("Found floating IP pool id using its name")
 	}
 
+	if d.TenantName != "" {
```

Comment:
If think it would be better to check `if d.TenantId == ""` rather than `if d.TenantName != ""` because it's common to have an OpenStack rc file containing both the tenant id and the tenant name. In such a situation it would be a pity to compute the id from the name.